packages/plugin-browserify/src/__snapshots__/plugin.test.ts.snap

```diff
@@ -6,11 +6,11 @@ exports[`plugin applies a transform 1`] = `
     function o(i, f) {
       if (!n[i]) {
         if (!e[i]) {
-          var c = \\"function\\" == typeof require && require;
+          var c = "function" == typeof require && require;
```

Comment:
It seems that Jest has changed the snapshot generator. All changes are like this though, it's not changing any actual code.